mobile_app/checkout_build_repos.py

```diff
@@ -0,0 +1,138 @@
+"""
+Checks out the various repositories needed to do a build
+based on a passed in configuration, switches them to the
+specified branch, and points the code at the config by writing
+an "edx.properties" file to the code repo.
+"""
+
+import git
+import json
+import logging
+import os
+import sys
+
+from mobile_app.path_constants import (
+    CODE_CHECKOUT,
+    CONFIG_CHECKOUT,
+    CONFIG_FILE,
+    PROPERTIES_FILE
+)
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def _write_properties(
```

Comment:
Nit: Per clean code, move the internal helper methods below the top-level methods.